src/Integration.Vsix.UnitTests/CFamily/ProtocolTest.cs

```diff
@@ -110,9 +99,9 @@ public void Read_Response()
         }
 
         [TestMethod]
-        public void Response_Filtering_NoFiltering()
+        public void Response_WithMultipleMessages_MultipleMessageAreReturned()
```

Comment:
This test is now testing that responses with multiple messages process all of the messages.